certificateshop-logic/pom.xml

```diff
@@ -13,18 +13,43 @@
         <artifactId>certificateshop</artifactId>
         <version>1.0-SNAPSHOT</version>
     </parent>
+    <properties>
+            <jackson-annotations.version>2.12.4</jackson-annotations.version>
+    </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-jdbc</artifactId>
+        </dependency>
+
         <!-- MySQL-->
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
         </dependency>
 
-        <!-- ConnectionPool -->
+<!--        &lt;!&ndash; ConnectionPool &ndash;&gt;-->
+<!--        <dependency>-->
+<!--            <groupId>com.zaxxer</groupId>-->
+<!--            <artifactId>HikariCP</artifactId>-->
+<!--        </dependency>-->
         <dependency>
-            <groupId>com.zaxxer</groupId>
-            <artifactId>HikariCP</artifactId>
+            <groupId>org.springframework.data</groupId>
+            <artifactId>spring-data-bom</artifactId>
+            <version>${spring-data-bom.version}</version>
+            <scope>import</scope>
+            <type>pom</type>
+        </dependency>
```

Comment:
You can't use spring data in a scope of this module, why did you add this dependency?

Reply:
there are some methods which out of the functionlal. Those methods use spring data. I studied spring data too. That is the point whz i use this dependency.

Reply:
You should remove this dependency and related functionality since this is not allowed in this module.